src/lib/validation.ts

```diff
@@ -30,7 +30,7 @@ type ValidatorMap = {
 };
 
 const TypeValidatorMap: ValidatorMap = {
-  dob: new ValidatorDateOfBirth(),
+  dob: new ValidatorAge(),
```

Comment:
here we have a `ValidatorAge` class exported from a file named `validator-dob` for a type called `dob`. Can we be consistent? Call everything Age or everything DoB?